### PR TITLE
Separate out CoreFX dependency subscriptions

### PR DESCRIPTION
**Pull request by dagood**
Now there are CoreFX (build pipeline), External (TFS) and CoreCLR (from its build pipeline).

Separating these out now because the CoreCLR build pipeline is turning on.

/cc @eerhardt @weshaggard @wtgodbe

### CODE REVIEW

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -95,7 +95,26 @@
...
+        // This handler will bring the Latest CoreCLR master build into CoreFX master
+        {
+          "maestroAction": "corefx-general",
+          "maestroDelay": "00:10:00",
+          "ScriptFileName": "UpdateDependencies.ps1",
+          "Arguments": [
+            "-GitHubUser dotnet-bot",
+            "-GitHubEmail dotnet-bot@microsoft.com",
+            "-GitHubPassword `$(`$Secrets['DotNetBotGitHubPassword'])",
+            "-CoreFxVersionUrl https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/coreclr/master/Latest.txt",
```
Comment (weshaggard) [inline Maestro/subscriptions.json]: Didn't this argument name change?

Comment (eerhardt) [inline Maestro/subscriptions.json]: Yes, and it looks like it has been breaking the maestro action:

https://devdiv.visualstudio.com/DevDiv/_build?favDefinitionId=1084&_a=summary&buildId=191519

Comment (dagood) [inline Maestro/subscriptions.json]: Thanks, completely slipped my mind to update usages of the param name. I'll change this PR to fix.

File: Maestro/subscriptions.json [hunk 2]
```diff
@@ -55,7 +55,7 @@
             "-GitHubUser dotnet-bot",
             "-GitHubEmail dotnet-bot@microsoft.com",
             "-GitHubPassword `$(`$Secrets['DotNetBotGitHubPassword'])",
-            "-CoreFxVersionUrl https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/corefx/release/1.0.0/Latest.txt",
+            "-VersionFileUrl https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/corefx/release/1.0.0/Latest.txt",
```
Comment (weshaggard) [inline Maestro/subscriptions.json]: Actually the release branch still has the old argument name for now so you sholudn't change it yet.

**Comment by weshaggard** [comment]
Other then the one comment release branch LGTM

File: Maestro/subscriptions.json (hunk 2 above)
Comment (dagood) [inline Maestro/subscriptions.json]: Fixed, thanks.

**Pull request merged**